src/main/main.java

```diff
@@ -23,27 +24,41 @@ public void init() {
     public void loop() { //Naufal the goat
```

Comment:
Would be nice if this comment deleted 😁

Reply:
Just merge it to main after, i would say

Reply:
@DispySK please guy!

Reply:
removed